### PR TITLE
Free Gish

### PR DESCRIPTION
**Pull request by Cebion**
Game Information
Title: Free Gish
URL: https://github.com/freegish/freegish

Controls

| Button | Action |
|--|--| 
|DPAD| Steer|
|A | Stick|
|B | Jump|
|X | Heavy|
|Y | Slide|
|L1/R1 | Y/N|
|L2 | C (Continue)|
|R2 | P (Practice)|

CFW Tests:
[X] AmberELEC
[X] ArkOS
[X] ROCKNIX
[X] MuOS
[X] Knulli (Optional)
[] Crossmix (Optional)

Architectures: 
[x] Aarch64
[X] Panfrost (Mainline)

Resolutions: 
[] 480x320 (Optional)
[X] 640x480
[X] 720x720 (RGB30) (Optional)
[X] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

**Review by tabreturn — COMMENTED** [review]
Is Yar's Revenge supposed to be bundled with this PR?

**Comment by Cebion** [comment]
> Is Yar's Revenge supposed to be bundled with this PR?

Whoops, fixed. That happens if you don't switch to branch main first :P

File: ports/freegish/port.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "version": 3,
+  "name": "freegish.zip",
+  "items": [
+    "Free Gish.sh",
+    "freegish"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Gish",
```
Comment (cdeletre) [inline ports/freegish/port.json:10]:
```suggestion
    "title": "Free Gish",
```

File: ports/freegish/gameinfo.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    <path>./Free Gish.sh</path>
+    <name>Free Gish</name>
+    <desc>Free Gish is a 2D platform game, where the player maneuvers character of a ball of tar. Character may become sticky, slick, heavy and can jump.
+The game contains only first seven levels of the single player campaign and few multiplayer levels for mini games like sumo or football.
+Freegish is based on open sourced code of the famous game called Gish with added free art assets.</desc>
+    <releasedate>20040510T000000</releasedate>
+    <developer>Chronic Logic </developer>
+    <publisher>Chronic Logic </publisher>
+    <genre>Arcade/Platformer/Puzzle</genre>
+    <image>./freegish/screenshot.jpg</image>
```
Comment (cdeletre) [inline ports/freegish/gameinfo.xml:13]: freegish-fs8
:)

**Review by cdeletre — CHANGES_REQUESTED** [review]

**Review by JanTrueno — APPROVED** [review]

**Review by JanTrueno — APPROVED** [review]

**Review by JanTrueno — APPROVED** [review]

**Pull request merged**